.clang-format

```diff
@@ -0,0 +1,49 @@
+BasedOnStyle: WebKit
+
+AllowShortBlocksOnASingleLine: true
+AllowShortFunctionsOnASingleLine: true
+
+BreakBeforeBraces: Custom
+BraceWrapping:
+  AfterClass:      true
+  AfterControlStatement: true
+  AfterEnum:       true
+  AfterFunction:   true
+  AfterNamespace:  false
+  AfterObjCDeclaration: false
+  AfterStruct:     true
+  AfterUnion:      true
+  AfterExternBlock: true
+  BeforeCatch:     true
+  BeforeElse:      true
+  IndentBraces:    false
+  SplitEmptyFunction: false
+  SplitEmptyRecord: false
+  SplitEmptyNamespace: false
+
+BreakBeforeBinaryOperators: NonAssignment
+FixNamespaceComments: true
+
+SortIncludes: true
+IncludeBlocks: Regroup
+IncludeCategories:
+  - Regex:           '<[[:alnum:]._]+>'
+    Priority:        1
+  - Regex:           '^(<|"(gtest|gmock|json|asio|network/uri|antlr))'
+    Priority:        2
+  - Regex:           '.*'
+    Priority:        3
+
+IndentCaseLabels: true
+NamespaceIndentation: None
+IndentPPDirectives: AfterHash
+KeepEmptyLinesAtTheStartOfBlocks: false
+MaxEmptyLinesToKeep: 3
+
+SpaceAfterTemplateKeyword: false
+Standard: Cpp11
```

Comment:
what about Cpp17?

Reply:
cpp17 is supported only since clang-format-10 and yields the same results as cpp11.